src/main/java/org/gradle/profiler/mutations/ClearGradleUserHomeMutator.java

```diff
@@ -7,16 +7,16 @@
 import java.io.UncheckedIOException;
 import java.nio.file.Files;
 
-public class ClearGradleUserHomeMutator extends AbstractCleanupMutator {
+public class ClearGradleUserHomeMutator extends AbstractScheduledMutator {
     private final File gradleUserHome;
 
-    public ClearGradleUserHomeMutator(File gradleUserHome, CleanupSchedule schedule) {
+    public ClearGradleUserHomeMutator(File gradleUserHome, Schedule schedule) {
         super(schedule);
         this.gradleUserHome = gradleUserHome;
     }
 
     @Override
-    protected void cleanup() {
+    protected void executeOnSchedule() {
```

Comment:
🤔 Note: This change will require also a change of some logic in `gradle/gradle` when we upgrade the profiler. I know about this one:
https://github.com/gradle/gradle/blob/9b00b5d6f111a3542959598fb31c5c7181c15972/testing/internal-performance-testing/src/main/groovy/org/gradle/performance/mutator/ClearArtifactTransformCacheWithoutInstrumentedJarsMutator.groovy#L45 

It's not a problem but we should be aware of that when we upgrade a profiler.